bedrock/firefox/views.py

```diff
@@ -572,6 +572,27 @@ def hello(request):
         {'video_url': videos.get(request.locale, videos.get('en-US'))})
 
 
+def hello_screen_sharing(version):
+    try:
+        if version.endswith('a1') or version.endswith('a2'):
```

Comment:
This is the only thing I felt unsure on. Dev edition is `a2` and Nightly `a1`, so as long as those don't change this will be ok. I wasn't sure if there's a more robust way though...